zjit/src/cruby_methods.rs

```diff
@@ -83,7 +173,13 @@ pub fn init() -> Annotations {
     annotate!(rb_cNilClass, "nil?", types::TrueClass, no_gc, leaf, elidable);
     annotate!(rb_mKernel, "nil?", types::FalseClass, no_gc, leaf, elidable);
 
+    // Annotate builtin functions using the new method-based approach
```

Comment:
I'm not sure if I follow what's "new" about it? If you mean it's new because it's just implemented, it may not hold for future readers.

To clarify that it's annotating on "the only" builtin function, I would write:

```suggestion
    // Annotate builtin functions for ISEQs that have only one invokeblock instruction
```

Reply:
Yeah I agree it's not too helpful in the future so I've deleted it.